Add deployment monitor to dbt cloud deploy page (#3199)
dbt-labs/docs.getdbt.com: website/docs/docs/dbt-versions/release-notes/08-May-2023/deployment-monitor.md

---

> title: "Use the Deployment monitor page for greater visibility to your job run health"

Reviewer: ```suggestion
title: "Use the Deployment Monitor page for greater visibility to your job run health"
```

id: "deployment-monitor"
description: "May 2023: "
sidebar_label: "New: Deployment monitor visualizes your run health"
sidebar_position: 5
tags: [scheduler]
---

> The new Deployment monitor helps you gain greater visibility into your deployment health and quickly detect any issues with the improved filtering and sorting capabilities. You can now easily curate the view based on environment, prioritizing the ones that matter most, such as Production. With pre-filtered lists for in progress and in queue activities, it's quicker and easier to track the relevant activity and stay on top of your deployment status.

Reviewer: ```suggestion
The new **Deployment Monitor** page helps you gain greater visibility into your deployment health and quickly detect any issues with the improved filtering and sorting capabilities. You can prioritize and customize the view based on the environment, such as **Production** from the dropdown menu at the top right of the page.

Use prefiltered **In Progress** and **In Queue** views to track relevant activities and stay up-to-date on your deployment status.
```

Author: nice one @nghi-ly ! thank you! the 'production' is title capped because i was referring to the deployment monitor UI, which returns the filter as **Production** - do you think that should still be lowercased? it does it for all environments 
![Screenshot 2023-05-31 at 17 03 58](https://github.com/dbt-labs/docs.getdbt.com/assets/89008547/9e0c5065-6351-4400-a0f9-991c253f8dcb)

Reviewer: oh, i see. you're right, **Production** is the correct font decoration if that was your intention. 

maybe for the sake of clarity, suggest clarifying the exact UI element you're referring to like "such as **Production** from .... " using location, UI component, or how to see it (get to it)

Author: good callout, made that tweak!

> The Deployment monitor is accessible by going to the main dbt Cloud dashboard page or clicking the dbt logo on the upper-left. Some of the new and enhanced features includes:

Reviewer: ```suggestion
The **Deployment Monitor** is accessible from the main dbt Cloud dashboard page or by clicking the dbt logo on the upper-left. Some of the new and enhanced features include:
```

- A new run history visualization component to quickly scan a larger number of recent runs and get a better overview of overall health/activity.

> - Pre-filtered lists have been added for In Progress, In Queue, and Latest Failed jobs. 

Reviewer: ```suggestion
- Prefiltered views added to display **In Progress**, **In Queue**, or **Latest Failed Runs**.
```

Author: maybe i should rephrase this so its clearer it describes the UI. so:

>Prefiltered views added to display **In Progress**, **In Queue**, or **Latest Failed** jobs.

![Screenshot 2023-05-31 at 17 05 51](https://github.com/dbt-labs/docs.getdbt.com/assets/89008547/834ae6be-54b6-4254-aa0b-9fe3b417b973)

Reviewer: oh i get it, sounds good!! sugg using "**Latest Failed Runs**" since that's the exact UI label

> - Prefiltered lists have been added to display **In Progress**, **In Queue**, and **Latest Failed** jobs. 

Author: ```suggestion
- Prefiltered views added to display **In Progress**, **In Queue**, or **Latest Failed Runs**.
```

- Filter the page by environment, with the preference being stored for future use. 

> - Lastly, the visualization at the top of the page now filters runs by trigger and displays the last 24 hours of runs, with a limit of 500.

Reviewer: ```suggestion
- The visualization at the top of the page now filters runs by trigger and displays the last 24 hours of runs, with a limit of 500.
```

> <Lightbox src="/img/docs/dbt-cloud/using-dbt-cloud/deploy-monitor.jpg" width="85%" title="The Deployment monitor dashboard displays overall deployment health and activity, allows you to filter the page by environment, and more" />

Reviewer: ```suggestion
<Lightbox src="/img/docs/dbt-cloud/using-dbt-cloud/deploy-monitor.jpg" width="85%" title="The **Deployment Monitor** dashboard displays overall deployment health and activity, allows you to filter the page by environment, and more" />
```
